examples/raw_input/RawInput.cpp

```diff
@@ -0,0 +1,82 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::filesystem::path resourcesDir()
+{
+#ifdef SFML_SYSTEM_IOS
+    return "";
+#else
+    return "resources";
+#endif
+}
+} // namespace
```

Comment:
Feel free to ignore but simply using `static` here would be less verbose:

```suggestion
static std::filesystem::path resourcesDir()
{
#ifdef SFML_SYSTEM_IOS
    return "";
#else
    return "resources";
#endif
}
```

Reply:
Tried it, `clang-tidy` will complain due to `misc-use-anonymous-namespace`. Also, this is the way the function has been defined in other examples, probably for the same reason.